src/components/Person.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Person(props) {
+  let partner = '';
+
+  if (props.sex === 'm') {
+    partner = 'wife\'s';
+  } else {
+    partner = 'husband\'s';
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is&nbsp;
+        {props.name}
+      </h2>
+      {props.age
+        ? (
+          <p className="Person__age">
+            I am&nbsp;
+            {props.age}
+          </p>
+        )
+        : null
+      }
```

Comment:
```suggestion
      {props.age && (
          <p className="Person__age">
            I am&nbsp;
            {props.age}
          </p>
        )}
```
LEt's use proper conditional rendering syntax instead of ternaries here